app/build.gradle

```diff
@@ -33,11 +33,15 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
 
         debug {
+            minifyEnabled true
```

Comment:
Do we really need to run proguard on the debug build? It would increase the debug build time.